ui/src/app/machines/components/ActionFormWrapper/CloneForm/CloneFormFields/CloneFormFields.test.tsx

```diff
@@ -65,4 +65,41 @@ describe("CloneFormFields", () => {
       actualActions.find((action) => action.type === expectedAction.type)
     ).toStrictEqual(expectedAction);
   });
+
+  it("applies different styling depending on clone selection state", async () => {
+    const machine = machineFactory({ system_id: "abc123" });
+    const state = rootStateFactory({
+      machine: machineStateFactory({
+        items: [machine],
+        loaded: true,
+        selected: [],
+        active: null,
+      }),
+    });
+    const store = mockStore(state);
+    const wrapper = mount(
+      <Provider store={store}>
+        <Formik
+          initialValues={{ interfaces: false, source: "", storage: false }}
+          onSubmit={jest.fn()}
+        >
+          <CloneFormFields />
+        </Formik>
+      </Provider>
+    );
+    const getTableClass = () =>
+      wrapper.find(".clone-table").at(0).prop("className");
+    // Table has unselected styling by default
+    expect(getTableClass()).toBe("clone-table not-selected");
```

Comment:
These could be slightly less fragile if they were:
`expect(getTableClass().includes("not-selected")).toBe(true);` and `expect(getTableClass().includes("not-selected")).toBe(false);`.

Reply:
Good point!